code/src/index.css

```diff
@@ -11,3 +9,85 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.date {
+  font-style: italic;
+}
+
+
+/*Forms*/
+
+.thoughts-form, .thoughts-card {
+  margin: 20px auto;
+  border: 2px solid black;
+  width: 500 px;
+  box-shadow: 5px 5px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+
+form input {
+  font-family: 'Trispace', sans-serif;
+  margin-top: 15px;
+  padding: 10px 10px 30px;
+}
+
+
+.thoughts-card {
+  word-wrap: break-word; 
+}
+
```

Comment:
Longer text messages spill outside the container even though you added break-word . Try to add the class which you have already named in your app but never used. (remember the lonely variable 😜 ) I am probably the worst CSS guide but  try this:  

.thoughts-message {
    width: 100%;
}